docs/pom.xml

```diff
@@ -57,7 +142,7 @@
                     <failOnMissingWebXml>false</failOnMissingWebXml>
                     <webResources>
                         <resource>
-                            <directory>target/_site/docs</directory>
+                            <directory>${project.build.directory}/_site</directory>
```

Comment:
generation of the site should happen in the target folder.
So all generated stuff is removed when we call `mvn clean` else, it will mix source code and generated code in the current folder